src/h5web/dataset-visualizer/DimensionMapper.tsx

```diff
@@ -1,86 +1,78 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import ReactSlider from 'react-slider';
+import { isNumber } from 'lodash-es';
```

Comment:
Type-guard to check if `value is number`. I could implement my own with `typeof value === 'number'` but it is already there in `lodash` :shrug: